src/julia.h

```diff
@@ -1520,7 +1520,7 @@ typedef struct _jl_task_t {
 } jl_task_t;
 
 JL_DLLEXPORT jl_task_t *jl_new_task(jl_function_t *start, size_t ssize);
-JL_DLLEXPORT void jl_switchto(jl_task_t *t);
+JL_DLLEXPORT void jl_switchto(jl_task_t **pt);
```

Comment:
We should perhaps make `jl_switchto` a thin wrapper for `jl_switchto_ref`, more to make the API easier to use than for backwards-compatibility. Could also use a comment stating the that argument is written to.